r/R/zzz.R

```diff
@@ -1,3 +1,11 @@
+# Select pyMyAMI version
+pymyami_version <- "2.1.0"
```

Comment:
need to change the update_pymyami.py script to point to zzz.R instead of pymyami.R